source/common/router/router.cc

```diff
@@ -1620,7 +1620,8 @@ bool Filter::convertRequestHeadersForInternalRedirect(Http::RequestHeaderMap& do
 }
 
 void Filter::doRetry() {
-  ENVOY_STREAM_LOG(debug, "performing retry", *callbacks_);
+  ENVOY_STREAM_LOG_EVENT(debug, "http_router_retry", "performing retry to cluster {}", *callbacks_,
```

Comment:
can we include some information about request' path? I understand that stream ID is kind of unique but not sure how we will get stream ID <> request mapping

Reply:
Talked offline - since not all requests are routed based on their path adding the information about request's path here would not be useful in the context of all requests that Envoy makes. Having this event emitted without request's path sounds just fine then.